text/0016-named-tracers.md

```diff
@@ -14,41 +14,49 @@ _Creating Tracers and Meters using a factory mechanism and naming those Tracers
 
 The mechanism of "Named Tracers and Meters" proposed here is motivated by following scenarios:
 
-* For a consumer of OpenTelemetry instrumentation libraries, there is currently no possibility of influencing the amount of the data produced by such libraries. Instrumentation libraries can easily "spam" backend systems, deliver bogus data or - in the worst case - crash or slow down applications. These problems might even occur suddenly in production environments caused by external factors such as increasing load or unexpected input data.
+**Expensive instrumentation**
 
-* If a library hasn't implemented [semantic conventions](https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/data-semantic-conventions.md) correctly or those conventions change over time, it's currently hard to interpret and sanitize these data selectively. The produced Spans or Metrics cannot be associated with those instrumentation libraries later.
+For an operator of OpenTelemetry, there is currently no possibility of influencing the amount of the data produced by reporting libraries. Reporting libraries can easily "spam" backend systems, deliver bogus data, or - in the worst case - crash or slow down applications. These problems might even occur suddenly in production environments caused by external factors such as increasing load or unexpected input data.
+
+**Reporting library identification**
+
+If a reporting library hasn't implemented [semantic conventions](https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/data-semantic-conventions.md) correctly or those conventions change over time, it's currently hard to interpret and sanitize these data selectively. The produced Spans or Metrics cannot be associated with the library which reported them later.
+
+**Conflicting instrumentation libraries**
+
+It is the eventual goal of OpenTelemetry that library vendors implement the OpenTelemetry API, obviating the need to auto-instrument their library. An operator should be able to disable built-in instrumentation even if the library does not provide an explicit configuration to disable instrumentation.
```

Comment:
This use case seems to fall under "Expensive Instrumenation" (or generally "excessive instrumentation"). The conflict would occur if in addition to the librarie's built-in instrumentation, there is also an auto-instrumentation or an integration that is hard-coded as activated. Only then, does it make sense to use reporter-information to disable/discard the spans produced by one of the conflicting reporters.

Reply:
Not true. I may want to disable the OpenTelemetry integration that is built into some database driver in order to provide my own integration if the built-in version is lacking in some way.

Reply:
To me, the paragraph _only_ has the use case of disabling the built-in instrumentation of some library. Would this be more clear?

>It is the eventual goal of OpenTelemetry that library vendors implement the OpenTelemetry API, obviating the need to auto-instrument their library. An operator should be able to disable the OpenTelemetry integration that is built into some database driver or other library and provide their own integration if the built-in telemetry is lacking in some way. This should be possible even if the developer of that database driver has not provided a configuration to disable telemetry.

@Oberon00 WDYT?